### PR TITLE
Initial Task_11 commit

### PR DESCRIPTION
**Pull request by Letushev**
- [Repo](https://github.com/Letushev/kottans_frontend/tree/master/Task_11)
- [Demo](https://letushev.github.io/kottans_frontend/Task_11/)

### CODE REVIEW

**Review by lempiy — COMMENTED** [review]
I believe its done very well. Can't even find what to blame for :1st_place_medal: .

Probably, you can skip using !important here:
https://github.com/Letushev/kottans_frontend/blob/master/Task_11/styles.css#L481

While your form is not responsive I suggest to delete viewport meta tag in HTML head, in order to make your form usable on mobile.
https://github.com/Letushev/kottans_frontend/blob/master/Task_11/index.html#L6

This part is OK, but it was simpler to use inline-block for counter.
https://github.com/Letushev/kottans_frontend/blob/master/Task_11/styles.css#L224
But as I see absolute also worked nicely for you.

Semantic element for such block is `<aside>`
https://github.com/Letushev/kottans_frontend/blob/master/Task_11/index.html#L288

Semantic element for such block is `<legend>`
https://github.com/Letushev/kottans_frontend/blob/master/Task_11/index.html#L200

**Comment by lempiy** [comment]
Duplicate of issue in your repo to let mentors know. ^

File: Task_11/index.html [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+
+          <input type="range" name="urgent" min="-10" max="10" step="5" value="0">
+
+          <ul>
+            <li>I got time</li>
+            <li><strong>Average</strong></li>
+            <li>Yesterday</li>
+          </ul>
+
+          <div class="box">
```
Comment (lempiy) [inline Task_11/index.html]: Semantic element for such block is `<legend>` .

File: Task_11/index.html [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+      </div>
+
+    </form>
+
+  </main>
+  
+  <footer>
+    
+    <div class="whatWeDo">
```
Comment (lempiy) [inline Task_11/index.html]: Semantic element for such block is  `<aside>`

Comment (viktor-yakubiv) [reply Task_11/index.html]: Why should it be a `<legend>`?

Comment (lempiy) [reply Task_11/index.html]: oh confused, sec

Comment (lempiy) [reply Task_11/index.html]: now its ok.!

Comment (Letushev) [reply Task_11/index.html]: Could you explain why `<aside>`?
It is a little bit tricky for me...

File: Task_11/index.html (hunk 1 above)
Comment (Letushev) [reply Task_11/index.html]: It is not so obvious for me as well.
I have to absolutely position this `<legend>` to the bottom (as it should be the first child of the `<fieldset>`). Is it okay?
Also I can't put the text "With an average timeline set up, our expert translator can take a reasonable amount of time perfecting tour translation" inside a `<p>` tag (`<p>` isn't allowed inside `<legend>`). Then what element should contain that text?
Thanks

**Review by myroslav-kolomiiets — APPROVED** [review]
Look's great! Nothing to add)

**Comment by turchak** [comment]
With resolution 3440*1440 it looks like that. You can use `max-width: [some-width]px` for some elements

**Pull request closed without merging**